x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -253,7 +255,7 @@ public void testJoinValidatorForFIPSLicense() throws Exception {
         new Security.ValidateLicenseForFIPS(false).accept(node, state);
 
         final boolean isLicenseValidForFips =
-            FIPS140LicenseBootstrapCheck.ALLOWED_LICENSE_OPERATION_MODES.contains(license.operationMode());
+            Security.FIPS_ALLOWED_LICENSE_OPERATION_MODES.contains(license.operationMode());
```

Comment:
This (existing) test is testing 2 different scenarios based on randomness in the license type.
Can we move it to 2 explicit test cases?